### ISSUE TITLE
CarpUser don't serialise OAuth token

### ISSUE THREAD

**Issue by bardram** [cph-cachet/carp.sensing-flutter#96]
In `carp_webservice` it seems like the OAuth token (in `CarpUser`) isn't serialized to JSON.

**Comment by bardram** [comment]
In `carp_user.dart` we have

```
  OAuthToken _token;

  /// The OAuth 2.0 [OAuthToken] for this user, once authenticated to CARP
  OAuthToken get token => _token;
```

which seems to be the problem.

**Comment by bardram** [comment]
Fixed in version `0.5.0`.
